### PR TITLE
Migration/l10n br account

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Comment by rvalyi** [comment]
:-1: 

So as I commented on https://github.com/OCA/l10n-brazil/pull/143 I think we should now get rid of the l10n_br_product module. I'm pushing again after updated Travis setup so we may have lint tests at least for now.

**Comment by rvalyi** [comment]
still many stuff to lint https://travis-ci.org/OCA/l10n-brazil/jobs/66159282

**Comment by rvalyi** [comment]
(so I also killed the l10n_br_product dependency in the last commit)

File: l10n_br_account/models/l10n_br_account.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+class L10n_brAccountCce(models.Model):
+    _name = 'l10n_br_account.invoice.cce'
+    _description = u'Cartão de Correção no Sefaz'
+
+    # TODO nome de campos devem ser em ingles
+    invoice_id = fields.Many2one('account.invoice', 'Fatura')
+    motivo = fields.Text('Motivo', readonly=True, required=True)
+    sequencia = fields.Char(
+        'Sequencia', help=u"Indica a sequencia da carta de correcão")
+    cce_document_event_ids = fields.One2many(
```
Comment (danimaribeiro) [inline l10n_br_account/models/l10n_br_account.py:47]: https://github.com/OCA/l10n-brazil/issues/95

File: l10n_br_account/models/l10n_br_account.py [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+
+class L10n_brAccountInvoiceCancel(models.Model):
+    _name = 'l10n_br_account.invoice.cancel'
+    _description = u'Cancelar Documento Eletrônico no Sefaz'
+
+    invoice_id = fields.Many2one('account.invoice', 'Fatura')
+    justificative = fields.Char(
+        'Justificativa', size=255, readonly=True, required=True,
+        states={'draft': [('readonly', False)]})
+    cancel_document_event_ids = fields.One2many(
```
Comment (danimaribeiro) [inline l10n_br_account/models/l10n_br_account.py:59]: https://github.com/OCA/l10n-brazil/issues/95

File: l10n_br_account/models/l10n_br_account.py [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+    status = fields.Char('Codigo', readonly=True)
+    message = fields.Char('Mensagem', readonly=True)
+    create_date = fields.Datetime(u'Data Criação', readonly=True)
+    write_date = fields.Datetime(u'Data Alteração', readonly=True)
+    end_date = fields.Datetime(u'Data Finalização', readonly=True)
+    state = fields.Selection(
+        [('draft', 'Rascunho'), ('send', 'Enviado'),
+         ('wait', 'Aguardando Retorno'), ('done', 'Recebido Retorno')],
+        'Status', select=True, readonly=True, default='draft')
+    document_event_ids = fields.Many2one(
```
Comment (danimaribeiro) [inline l10n_br_account/models/l10n_br_account.py:117]: https://github.com/OCA/l10n-brazil/issues/95

File: l10n_br_account/models/l10n_br_account.py [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+        u'Número Final', readonly=True,
+        states={'draft': [('readonly', False)]}, required=True)
+    state = fields.Selection(
+        [('draft', 'Rascunho'), ('cancel', 'Cancelado'),
+         ('done', u'Concluído')], 'Status', required=True, default='draft')
+    justificative = fields.Char(
+        'Justificativa', size=255, readonly=True,
+        states={'draft': [('readonly', False)]}, required=True)
+    invalid_number_document_event_ids = fields.One2many(
+        'l10n_br_account.document_event', 'document_event_ids', u'Eventos')
```
Comment (danimaribeiro) [inline l10n_br_account/models/l10n_br_account.py:312]: https://github.com/OCA/l10n-brazil/issues/95

**Comment by rvalyi** [comment]
So para informar, ontem o Renato fez um rebase e push -f mas esqueceu de fazer pull nessa branch. Dai eu re-appliquei os meus commits de PEP-8 lint e clean up de novo (com -f tb). Ainda temos que resolver o problema usptream no test do fiscal rule que apareceu depois que foram modificados os scripts Travis alguns meses atras ja.

**Comment by renatonlima** [comment]
Eu analisei a branch 8.0 do Odoo-brasil integrei as melhorias feitas la no módulo l10n_br_account, já é possível fazer o merge, @danimaribeiro após fazer este PR você pode fazer um PR para corrigir o que você comentou no #95? acredito que só ficou isso para a wave 2.

**Comment by rvalyi** [comment]
Hello, so I checked the branch and it's ok for me too. It has been green or nearly green for a while here now with few people involved, so I'm merging this in order to move forward and we will also consider subsequent PR for this module in case we missed anything. For now I think the priority is to get this merged and rebase depdent migration branches and get their upstream (fiscal rules) migrated and green too.

**Comment by danimaribeiro** [comment]
:+1: Yes, we definitely need to fix #95, I have a branch with the fix.
I'll send a PR.
Whats the correct branch for account-fiscal-rule?

**Pull request merged**